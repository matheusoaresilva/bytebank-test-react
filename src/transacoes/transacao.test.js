import React from 'react'
import { render } from '@testing-library/react'
import Transacao from './Transacao'

describe('transition', () => {
  it('match snapshot', () => {
    const { container } = render(<Transacao 
    data="08/09/2020"
    tipo="saque"
    valor="20.00"
    />);

    expect(container.firstChild).toMatchSnapshot();
  });
});