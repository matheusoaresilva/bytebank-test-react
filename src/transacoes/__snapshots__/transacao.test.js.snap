// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transition match snapshot 1`] = `
<div
  class="transacao-container"
>
  <p>
    08/09/2020
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
